docs/sym.sh

```diff
@@ -141,7 +137,6 @@ ensurePython38() {
 installWithPipx() {
   ensurePython38
```

Comment:
it looks to me like `ensurePython38` also could trigger a brew install, should there be some other logic there?

Reply:
It'd trigger if there is no pyenv. I guess the only alternative (other than trying to use `apt-get` or the like) if there is no pyenv, is to install pyenv. Is that better than using Homebrew? What do you think?

Reply:
We could just bail and tell the user to install Python, too.

Reply:
I think its fine for now but maybe we confirm with the user before continuing

Reply:
Just to clarify, does this sound right?

1. Check for Python 3.8. If it exists, return success.
2. Check for pyenv. If it exists, install Python 3.8 using pyenv, and return success if successful.
3. Check for Homebrew. If it exists, prompt the user to confirm installing Python 3.8 via Homebrew, and return success if the user says yes and the installation succeeded.
4. Return failure.

Reply:
yup!